packages/documentation/src/content/docs/admin/admin-user-guide.mdx

```diff
@@ -0,0 +1,274 @@
+
+:::note[What if I need to edit or delete a wallet address?]
+When managing wallet addresses in Rafiki, there are certain restrictions and limitations to be aware of:
+
+**Editing wallet address URLs**
+
+The URL of an existing wallet address cannot be edited. Changing the URL could disrupt the transaction history associated with that wallet address, potentially leading to inaccuracies in payment records.
+
+**Deleting wallet addresses**
+
+Wallet addresses cannot be deleted from the system. This restriction exists to ensure that any payments tied to a wallet address remain intact and accessible for reporting purposes.
+
+**Solution**
+
+In both cases, the recommended approach is to create a new wallet address and deactivate the old one. To deactivate a wallet address, change the wallet status to Inactive.
+:::
+
+## Webhooks
+
+The Webhook Events page allows you to monitor and manage webhook events within your Rafiki instance. Webhook events in Rafiki are the main communication channel between you and your Rafiki instance. See Webhook events for more information about webhook events.
+
+All webhook events that have been triggered are displayed in the table. For each webhook event, you can see the webhook ID, the event type, and the date and time of the event. There is also a field at the top of the page allowing you to filter the table by event type, making it easier to drill down into specific events.
+
+<LargeImg
+  src='/img/admin-guide/webhooks.png'
+  alt='A screenshot of the Rafiki Admin application user interface displaying the "Webhook Events" section. It shows a table containing one webhook event with details including ID, type of webhook, and the date of the event. A "View data" button is also present in the table row for a detailed JSON view of the event data.'
+/>
+
+To view the webhook event as a JSON representation, select **View data**.
+
+## Payments
+
+The Payments page allows you to view all incoming and outgoing payments in your Rafiki instance. This allows you to monitor payment activity as well as track the status of payments.
+
+All payments are displayed in the table. For each payment, you can view the unique payment ID, type of payment, state of the payment, and the date and time the payment was created. There is also a field at the top of the page allowing you to filter the table by payment type, making it easier to drill down into specific transactions.
+
+<LargeImg
+  src='/img/admin-guide/payments.png'
+  alt='A screenshot of the Rafiki Admin application user interface displaying the "Payments" section. It shows a table where payment details would be displayed including ID, payment type, payment state, and the date of the payment.'
+/>
+
+## Account settings
+
+The Account Settings page allows you to manage your personal account information, including updating your email address and password.
+
+<LargeImg
+  src='/img/admin-guide/payments.png'
```

Comment:
```suggestion
  src='/img/admin-guide/account-settings.png'
```

This points to the wrong image as well. So maybe we can fix that too. I suspect it's meant to go to account-settings.png